CMake/External_Darknet.cmake

```diff
@@ -0,0 +1,89 @@
+set(Darknet_DEPENDS)
+
+if(fletch_ENABLE_OpenCV)
```

Comment:
I think I asked this before, but don't recall .. is there a good reason not to just add OpenCV to darknet when OpenCV is enabled?